tests/testthat/test_pipeop_histbin.R

```diff
@@ -0,0 +1,25 @@
+context("PipeOpHistBin")
+
+task = mlr_tasks$get("iris")
+op = PipeOpHistBin$new()
+expect_pipeop(op)
+result = op$train(list(task))
```

Comment:
do the above in a test_that, otherwise the test fail reporting is broken I think. if you need it in multiple tests do a function (or in this case just copy paste since it is v little code)